shared/tracker2/bio/index.tsx

```diff
@@ -177,6 +180,24 @@ const Bio = (p: Props) => (
         {p.sbsDescription}
       </Kb.Text>
     )}
+    {p.blocked ? (
+      <Kb.Text type="BodySmallError" center={true} style={styles.blockedBackgroundText}>
+        <Kb.Text type="BodySmallError" center={true} style={styles.text} selectable={true}>
+          You blocked them.
+        </Kb.Text>
+        <Kb.Text type="BodySmallError" center={true} style={styles.text} selectable={true}>
+          {p.username} won’t be able to chat with you or add you to teams.
+        </Kb.Text>
+      </Kb.Text>
+    ) : (
+      p.hidFromFollowers && (
+        <Kb.Text type="BodySmallError" center={true} style={styles.blockedBackgroundText}>
+          <Kb.Text type="BodySmallError" center={true} style={styles.text} selectable={true}>
+            You hid them from your followers.
```

Comment:
what if both?

Reply:
We show the blocked one, as Cecile responded to me on the design.